SPECS-EXTENDED/openrdate/openrdate.spec

```diff
@@ -0,0 +1,111 @@
+Summary:        Good-old rdate date and time-setting software
+Name:           openrdate
+Version:        1.2
+Release:        23%{?dist}
+License:        BSD AND BSD WITH advertising
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://sourceforge.net/projects/openrdate
+Source0:        http://downloads.sourceforge.net/%{name}/%{name}-%{version}.tar.gz
+Patch0:         openrdate_sysctl.patch
+BuildRequires:  gcc
+BuildRequires:  make
+# openrdate-1.2-1 replaces rdate-1.4-24, which does not have active upstream
+Provides:       rdate = %{version}-%{release}
+Obsoletes:      rdate < 1.4-25
+
+%description
+Good-old date- and time-setting rdate software implementing RFC 868
+(inetd time) and RFC 2030 (SNTP/NTP) protocols. An independent package
+of OpenBSD's rdate program.
+
+%prep
+%setup -q
```

Comment:
I think we should consider `autosetup` here.

Reply:
> I think we should consider `autosetup` here.

I have implemented your review comments. Please look at the latest commit. Also ran buddy builds after that. Here are the results. 
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=302968&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=302969&view=results)